Validation.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PhoneBookApp
```

Comment:
You can use stand-alone namespaces to optimize the use of space : 

![image](https://user-images.githubusercontent.com/108711476/202898628-c9d64c5a-cefa-4c93-9c94-216762d6df04.png)